Stratis.Bitcoin/Consensus/ConsensusValidator.cs

```diff
@@ -14,56 +14,89 @@
 
 namespace Stratis.Bitcoin.Consensus
 {
-	public class ConsensusValidator
+	/// <summary>
+	/// A set of options with default values of the Bitcoin network
+	/// This can be easily overridable for alternative networks (i.e Stratis)
+	/// Capital style param nameing is kept to mimic core
+	/// </summary>
+	public class ConsensusOptions
 	{
-		NBitcoin.Consensus _ConsensusParams;
-		const int MAX_BLOCK_WEIGHT = 4000000;
-		public const int WITNESS_SCALE_FACTOR = 4;
-		public const int SERIALIZE_TRANSACTION_NO_WITNESS = 0x40000000;
+		public int MAX_BLOCK_WEIGHT = 4000000;
```

Comment:
Also some some identical constants in NStratis, but unfortunately they are private.
```
		private static readonly uint MAX_BLOCK_SIZE = 1000000;
		private static readonly ulong MAX_MONEY = 21000000ul * Money.COIN;
```
Would be good to get all the magic number declarations in one place.